SPECS/perl/perl.spec

```diff
@@ -315,11 +363,13 @@ Suggests:       perl-doc = %{perl_version}-%{release}
 # We need this to break the dependency loop, and ensure that perl-libs 
 # gets installed before perl-interpreter.
 Requires(post): perl-libs
-# Same as perl-libs. We need macros in basic buildroot.
-Requires(post): perl-macros
-
-Provides: /bin/perl
 
+# suidperl isn't created by upstream since 5.12.0
+Obsoletes:      perl-suidperl <= 4:5.12.2
+# perl was renamed to perl-interpreter and perl-core renamed to perl
+# <https://fedoraproject.org/wiki/Changes/perl_Package_to_Install_Core_Modules>,
+# bug #1464903.
+Obsoletes:      perl < 4:5.26.0-395
```

Comment:
I think we can remove the `Obsoletes` for Mariner, since we're starting fresh in 2.0.

```suggestion
```